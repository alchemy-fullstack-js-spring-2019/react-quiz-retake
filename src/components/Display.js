import React from 'react';
import PropTypes from 'prop-types';

function Display({ headerText, footerText, image }) {

  const styles = {
    width: '100px'
  }

  return (
    <>
      <p>{headerText}</p>
      <img src={image} styles={styles} />
      <p>{footerText}</p>
    </>
  )

}

Display.propTypes = {
  headerText: PropTypes.string.isRequired,
  footerText: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default Display;
