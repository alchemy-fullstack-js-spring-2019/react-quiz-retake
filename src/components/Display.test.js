import React from 'react';
import { shallow } from 'enzyme';
import Display from './Display';

describe('Display', () => {
  it('Renders display', () => {
  const wrapper = shallow(<Display headerText="ugh" image="/img" footerText="no" />);
    expect(wrapper).toMatchSnapshot();
  });
});
