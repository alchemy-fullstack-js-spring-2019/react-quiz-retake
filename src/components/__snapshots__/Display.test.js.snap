// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Renders display 1`] = `
<section>
  <h2>
    ugh
  </h2>
  <img
    style={
      Object {
        "height": "250px",
        "width": "250px",
      }
    }
  />
  <h2>
    no
  </h2>
</section>
`;
